### ISSUE TITLE
Deploy image without tag

### ISSUE THREAD

**Issue by jvanz** [shipyard/shipyard#583]
Hi, folks

I'm testing Shipyard and I have a doubt. When I deploy an image (e.g. ubuntu) without tag I can see this:
![docker_img](https://cloud.githubusercontent.com/assets/1514798/9474962/87788116-4b18-11e5-9f4a-ba1162c6ee5b.png)

After finished these processes I can see a lot of ubuntu images. Why? Should  Shipyard/Citadel download only the latest tag?

**Comment by ehazlett** [comment]
Hmm perhaps it's a bug.  I would recommend to check the "swarm-v3" branch
as that is where the development has continued.  We are no longer
developing citadel.  Swarm v3 is planned to be merged hopefully soon.  You
can see testing docs at http://v3-docs.shipyard-project.com/.

**Comment by jvanz** [comment]
Nice! Do you have some ideia when the version 3 will be released? What are missing?

**Comment by jvanz** [comment]
Hi!
I'm trying to build the Shipyard but I have some issues in the process. I have tried to build using the

**Comment by jvanz** [comment]
I'm trying to build the Shipyard swarn-v3 but it is not working for me. I have tried to build using the `script/build` script. But when the script try to run the container to build the Shipyard some issues with godeps occurs. See:

<script src="https://gist.github.com/jvanz/9e862a48b2b50a193e29.js"></script>

PS: I also tried to run `make build` inside controller directory. But the same problem occurs... Am I missing something important?
